syntax = "proto3";

package rpc;

option cc_generic_services = true;

message AddRequest {
  int32 a = 1;
  int32 b = 2;
}

message AddResponse {
  int32 result = 1;
}

service AddService {
  rpc Add(AddRequest) returns(AddResponse);
}